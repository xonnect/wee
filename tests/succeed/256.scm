;; to test fixnum boundary conditions

(define x 256)
(define y 257)

(display (> x y))
(display (< x y))
(display (> x 255))
(newline)
(display (+ x y))
(newline)
(display (+ x 1))
(display (= (+ x 1) y))
(newline)
(display (- y 1))
(display (= x (- y 1)))
(newline)
(display (- x 1))
(newline)
