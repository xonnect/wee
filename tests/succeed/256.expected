#f#t#t
513
257#t
256#t
255
